docs/releasenotes/8.1.0.rst

```diff
@@ -44,8 +36,29 @@ already exists for the ICNS format.
 Security
 ========
 
-An out-of-bounds read when saving TIFFs with custom metadata through libtiff has been
-fixed, as well as when saving a GIF of 1px width.
+This release includes security fixes.
+
+* An out-of-bounds read when saving TIFFs with custom metadata through libtiff
+* An out-of-bounds read when saving a GIF of 1px width
+* :cve:`CVE-2020-35653` Buffer Read Overrun in PCX Decoding.
+
+The PCX Image decoder used the reported image stride to calculate the row buffer,
+rather than calculating it from the image size. This issue dates back to the PIL fork.
+Thanks to Google's OSS-Fuzz project for finding this.
```

Comment:
We could link "Google's OSS-Fuzz project" to https://github.com/google/oss-fuzz, here and below.

Reply:
Ok, done